packages/frontend/src/pages/InferenceServerDetails.svelte

```diff
@@ -192,10 +199,15 @@ export function goToUpPage(): void {
               <div class="bg-[var(--pd-content-card-bg)] rounded-md w-full px-4 pt-2 pb-4 mt-2">
                 <span class="text-sm text-[var(--pd-content-card-text)]">Server</span>
                 <div class="flex flex-row gap-4">
-                  <div
-                    class="bg-[var(--pd-label-bg)] text-[var(--pd-label-text)] rounded-md p-2 flex flex-row w-min h-min text-xs text-nowrap items-center">
-                    http://localhost:{service.connection.port}/v1
-                  </div>
+                  {#if service.status === 'running' && service.type === InferenceType.LLAMA_CPP}
```

Comment:
I think that the value of the URL should be computed by the Inference server implementation (llama_cpp,...) as the /docs suffix is not generic. And return null will condition the presence of the button without llama_cpp specific code here

Reply:
I don't see an easy way of doing this, as the LlamaCpp Inference provider is a worker, not a manager that can be reused

Reply:
Maybe the connection field should be expanded to contain not only the port

Reply:
The value of the field is now the responsability of the Inference Provider!